example/lib/components/position_component.dart

```diff
@@ -9,12 +9,12 @@ class Position {
 }
 
 class PositionComponent extends Component<Position> {
-  double x;
+  double? x;
 
-  double y;
+  double? y;
```

Comment:
should these be late? it seems they always start with 0 at least

Reply:
`late` cant be nullified, and we do need to clear up the memory when the component gets disposed.